### PR TITLE
[FEAT] Lobby Page Restyle

### LINKED ISSUE #298: [FEAT] Modify the styling for frontend components
## Feature Description: 
- Modify the styling of the frontend components so that the app has a dark theme and a cleaner aesthetic
- Remove the Comic sans fonts and the cartoon aesthetic
- Make use of Bootstrap components for layout and styling

The changes will be made in frontend/components

## Blocked by:
#17

### PR DESCRIPTION
**Pull request by reeved**
Co-authored and Pair-Programmed with @SreenikethRaghavan on this.

- Revamped the lobby page with dark theme and responsiveness
- Re-did the user avatars and removed backgrounds from them. They are now in /public
- Minor Restyle to countdown page
- Minor changes on other pages and components too

This is the final PR to fix #298.

After this, the whole app will be restyled :D 

Before:
image

After:
image

### CODE REVIEW

**Review by SreenikethRaghavan — APPROVED** [review]
Reviewed the code on a discord call with @reeved. I was the navigator for the pair programming session and the changes LGTM 👍

File: frontend/src/components/4.Lobby/Lobby.js [hunk 1]
```diff
@@ -71,25 +70,29 @@ const Lobby = (props) => {
...
+        <Card className='cardMain'>
+          <Card.Img
+            variant='top'
+            src={`avatars/${i}.png`}
+            className='cardImage'
+          />
+          <Card.Body>
+            <Card.Title>{users[i]}</Card.Title>
+          </Card.Body>
+        </Card>
```
Comment (Dhors) [inline frontend/src/components/4.Lobby/Lobby.js:86]: Nice changes reeves,
The new restaurant cards look so much nicer!

**Review by Dhors — APPROVED** [review]
Nice changes reeve,

The consistency in the resizable font and the buttons throughout the system makes yumble look so much more interconnected.

Nice reducing duplicate code with the help.js file. Just one small thing I noticed, but I feel we could possibly change the name of the help.js file to avoid confusion, I feel since its used for help and share now. Although this isn't that important so you can just ignore it.

**Comment by reeved** [comment]
> Nice changes reeve,
> 
> The consistency in the resizable font and the buttons throughout the system makes yumble look so much more interconnected.
> 
> Nice reducing duplicate code with the help.js file. Just one small thing I noticed, but I feel we could possibly change the name of the help.js file to avoid confusion, I feel since its used for help and share now. Although this isn't that important so you can just ignore it.

Agreed. There is still repetition of the modal from the map and pie chart modals. I'll create an issue about refactoring them into one component that is multi-purpose :D

**Pull request merged**